lib/src/method_channel/convert/map_padding.dart

```diff
@@ -0,0 +1,25 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import '../../types/types.dart';
+import '../method_channel.dart';
+
+/// [MapPadding] convert extension.
+/// @nodoc
+extension ConvertMapPadding on MapPadding {
```

Comment:
Do not add extension to EdgeInsets here.
instead could we have factory MapPaddingDto class 
"fromEdgeInsets"

Reply:
Wouldn't we have to have this factory where the MapPaddingDto is implemented? That would be the pigeon file and pigeon can't import other flutter files so we couldn't use the EdgeInsets there.

Reply:
Current approach is OK